pytest_plugins/metadata_markers.py

```diff
@@ -24,6 +25,10 @@ def pytest_addoption(parser):
         '--assignee',
         help='Comma separated list of assignees to include in test collection',
     )
+    parser.addoption(
```

Comment:
We dont need the BaseOS to filter the test collection because we dont have tests/components that work on satellite installed on Base RHEL8 but not on Base RHEL7. It's like, all tests should work on RHEL7 and RHEL8 so there is no choice to choose the tests based on BaseOS. 

So just remove this pytest option!

Reply:
@jyejare I added this as an option to filter out the the tests based on RHEL version if there will be different tests for RHEL7 and RHEL8 in coming satellite versions but since we don't have that use case in near future then I can remove the marker and just keeping the user_properties.